docs/guides/_include/zbm-setup.rst

```diff
@@ -2,11 +2,30 @@ Install and configure ZFSBootMenu
 ---------------------------------
 
 Set ZFSBootMenu properties on datasets
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Assign command-line arguments to be used when booting the final kernel. Because ZFS properties are inherited, assign the
-common properties to the ``ROOT`` dataset so all children will inherit common arguments by default.
+.. tabs::
 
-.. code-block::
+  .. group-tab:: Encrypted
 
-  zfs set org.zfsbootmenu:commandline="quiet" zroot/ROOT
+      Assign command-line arguments to be used when booting the final kernel. Because ZFS properties are inherited,
+      assign the common properties to the ``ROOT`` dataset so all children will inherit common arguments by default.
+
+      .. code-block::
+
+        zfs set org.zfsbootmenu:commandline="quiet" zroot/ROOT
+
+      Setup key caching in ZFSBootMenu.
+
+      .. parsed-literal::
+
+        zfs set org.zfsbootmenu:keysource="zroot/ROOT/\ |distribution|"
```

Comment:
This is missing the target encroot where the property will be set.

Reply:
Whoops. I now have it set on `zroot`, which is where the encroot actually is configured for these guides.